### ISSUE TITLE
Transpile all files with babel?

### ISSUE THREAD

**Issue by azu** [azu/espower-babel#9]
Currently, espower-babel transpile only `test/*.js` and this behavior cause confusion.
So, I think that espower-babel should transpile all files without node_modules.

But, I worried about unexpected transpiled result.
Babel treat the target code as ES6 modules.
https://twitter.com/azu_re/status/592995278715650048

---
## Plan A
- Transpile all files by default
- The user can config by [babelrc · Babel](http://babeljs.io/docs/usage/babelrc/).
## Plan B
- espower-babel is deprecated.
- The user use [twada/babel-plugin-espower](https://github.com/twada/babel-plugin-espower) directly.
## Plan C
- ???

**Comment by twada** [comment]
> Babel treat the target code as ES6 modules.
> https://twitter.com/azu_re/status/592995278715650048

Using

`babel --blacklist strict script.js`

OR

``` javascript
require("babel").transform("code", { blacklist: ["strict"] });
```

would be a solution? (especially, for Plan A users?)

ref: [Why is this being remapped to undefined? FAQ · Babel](https://babeljs.io/docs/faq/#why-is-this-being-remapped-to-undefined-)

**Comment by azu** [comment]
@twada Thanks. But espower-babel should **not** apply that options automatically.
The user's config get preference over espower-babel's config.

How to remove `--blacklist` options If espower-babel apply `--blacklist strict` automatically **and** the user not want to apply `--blacklist strict`?
So, `--blacklist` options should be optional, I think.(Such options will be configured by babelrc)

**Comment by twada** [comment]
> But espower-babel should not apply that options automatically.

Agreed. espower-babel should not do so.

> So, --blacklist options should be optional, I think.(Such options will be configured by babelrc)

Ditto. Should be optional.

**Comment by azu** [comment]
implemented #10
